otx/algorithms/common/utils/data.py

```diff
@@ -33,9 +34,8 @@
     DirectoryPathCheck,
     check_input_parameters_type,
 )
-from otx.mpa.utils.logger import get_logger
 
-logger = get_logger()
+logger = get_logger(name="otx")
```

Comment:
How about changing `otx` to `otx.algorithm.common.utils.data`?

Reply:
Reflected, please review again.